src/modules/authentication/passport/oidc.strategy.ts

```diff
@@ -26,11 +26,36 @@ export function extractStepUpLevelFromJWT(jwt: string | undefined): StepUpLevel
     if (!jwt) {
         return StepUpLevel.NONE;
     }
-
     const decoded: CustomJwtPayload | null = decode(jwt) as CustomJwtPayload | null;
     return decoded?.acr ?? StepUpLevel.NONE;
 }
 
+export function getLowestStepUpLevel(): StepUpLevel {
+    return StepUpLevel.SILVER;
+}
+
+// timeout in seconds
+export function isStepUpTimeOver(req: Request, timeout: number): boolean {
+    const currentTime: number = new Date().getTime();
+    if (!req.session?.lastRouteChangeTime) return false;
```

Comment:
Could use `Date.now()`, but doesn't really matter (Performance is a tiny bit better)